php/editor.php

```diff
@@ -31,6 +31,7 @@ function enqueue_code_editor( $type, $extra_atts = [] ) {
 
 	$default_atts = [
 		'mode'          => $modes[ $type ],
+		'inputStyle'    => 'textarea',
```

Comment:
I had to add this option (default seems to be `contentEditable`) because there appears to be some conflicts regarding the cursor and the vim keymap.

Also, there was a strange behavior (It doesn't seem to be present in the previous Code Snippets version) where selections would look like this
![image](https://user-images.githubusercontent.com/39108635/161407325-7910ced7-11e1-4ed7-9741-8d5cc9f72a7f.png)

And using `'inputStyle' => 'textarea'` fixed it
![image](https://user-images.githubusercontent.com/39108635/161407238-361c7946-fecd-4fad-ab6f-71b513da3a1f.png)

The only caveat is that the default textarea seems to have a `rows=200` inline attribute that makes it super tall, and it will flash before it is initialized. It's probably something wordpress-related, but perhaps it can be fixed via css or some other way so it doesn't yank the layout.

I didn't do much digging on this, but I could if you think it's worth fixing.